src/views/Home/components/CLWCreditsDaysLeft/index.js

```diff
@@ -11,194 +11,192 @@ import './CLWChart.css';
 
 import { Card, CardHeader, CardContent, Typography, Grid, Button } from '@material-ui/core';
 
-export default class CLWCreditsDaysLeft extends Component {
-  constructor(props) {
-    super(props);
-
-    this.loadData = this.loadData.bind(this);
-
-    this.state = {
-      daysLeft: [],
-      chapelCredits: {},
-      error: null,
-      loading: true,
-      currSessionDescription: '',
+const CLWCreditsDaysLeft = () => {
+  const [state, setState] = useState({
+    daysLeft: [],
+    chapelCredits: {},
+    error: null,
+    loading: true,
+    currSessionDescription: '',
+  });
+
+  useEffect(() => {
+    const loadData = async () => {
+      setState({ loading: true });
+      try {
+        const daysLeftPromise = session.getDaysLeft();
+        const chapelCreditsPromise = user.getChapelCredits();
+        const currSessionPromise = session.getCurrent();
+        const daysLeft = await daysLeftPromise;
+        const chapelCredits = await chapelCreditsPromise;
+        const currSession = await currSessionPromise;
```

Comment:
We can streamline this data fetching code. It was written when `async await` syntax was still relatively new and our developers were unfamiliar with it. The more idiomatic (and clear) syntax would simply be:
```suggestion
const daysLeft = await session.getDaysLeft();
const chapelCredits = await user.getChapelCredits();
const currSession = await session.getCurrent();
```